### DISCUSSION TITLE
Sending Mass Emails

### DISCUSSION THREAD

**Discussion by haldonclark** [cerb/cerb-release#70] [Q&A] [answered]
Is it possible to send mass emails like a promotional email to a selected group of contacts in such a way that each recipient is the only recipient in the "to" field. On other words, you are not sending 1 message to 40 recipients (and each one is seeing all the other recipients), but you are sending 40 messages to 1 recipient each.

If this is possible, starting from which version and how?

**Comment by jstanden** [comment, marked as answer]
Hi @haldonclark,

We have a 'Broadcast' feature that you can use from bulk update from a worklist of various record types: tickets, email addresses, contacts, orgs, custom records, etc.

That will send a separate message to each recipient (they will be the only `To:`). You can also use placeholders to personalize each message.

That has been possible for years, so any relatively recent version will support it.

1. Click the search icon in the top right and choose a record type (e.g. Email Addresses)
2. Add search filters to create your mailing list.
3. Click the 'Bulk Update' icon.
4. Check the box for the 'Send Broadcast' section.
5. Choose the recipients (you can include various email-related fields and custom fields), subject, content, attachments, and status.
6. Either 'send now' or have it create drafts for each message that you can finalize.

**Reply by haldonclark** [reply]
Great!

If you are bored any time in the future, you could add the "deliver later" function. This would in fact enable sending scheduled emails.
I wish you a happy, healthy and prosperous 2024.

**Reply by jstanden** [reply]
Sure! That has come up a few times and it would be really useful. Particularly on a day like last Friday when we'd schedule messages to arrive Tuesday morning after the long holiday weekend. 😀

I'll bump the priority on the request for you: https://github.com/jstanden/cerb/issues/1021
